mobile/src/screens/chat-list/chat-list.tsx

```diff
@@ -43,14 +44,17 @@ const ChatList: React.FC = () => {
   const chatsLength = chats.length;
 
   const handleSelectChat = useCallback(
-    (title: string) => {
-      navigation.navigate(ChatScreenName.CHAT, { title });
+    (title: string, id: string | undefined) => {
+      navigation.navigate(ChatScreenName.CHAT, { title, id: id ?? '' });
     },
     [navigation],
   );
 
   const handleRedirectToChat = useCallback(() => {
-    navigation.navigate(ChatScreenName.CHAT, { title: 'New Chat' });
+    navigation.navigate(ChatScreenName.CHAT, {
+      title: 'New Chat',
+      id: '',
```

Comment:
```suggestion
```

Reply:
I need to pass id, it's mandatory; without it, I get an error

Reply:
it's ok in this case